testsuite/mpitests/test_sinusoidal_poisson_generator_5.sli

```diff
@@ -104,15 +104,10 @@ See also: test_sinusoidal_poisson_generator_{1,2,3,4,6}, test_sinusoidal_poisson
   % get events, replace vectors with SLI arrays
   % keep only non-empty arrays; empty ones are from off-process parrots
   (ALL LINES SHOULD BE EQUAL) ==  
-  sdets { [/events /times] get cva } Map { empty not exch ; } Select
+  sdets { [/events /times] get cva } Map { empty not exch ; } Select dup /spike_times Set
   { == } forall
```

Comment:
This line just prints all the spike times, filling the log with potentially garbled data from the two processes. I would remove it; then, you don't need the `dup /spike_times Set`.